libs/src/api/serviceProvider.js

```diff
@@ -69,9 +93,12 @@ class ServiceProvider extends Base {
   async autoSelectCreatorNodes (
     numberOfNodes = 3,
     whitelist = null,
-    blacklist = null
+    blacklist = null,
+    enforceSyncStatus = true
```

Comment:
Added this flag because previously autoSelect would only return Primaries that have syncStatus of isConfigured (have previously stored user data) and notBehind (are up to date with discprov's user state). we now use this autoSelect function in the sanityChecks.rolloverNodes() to backfill secondaries, treating the value returned for primary as the secondary to backfill. so this logic needed to change - for secondaries we don't care about isConfigured or isBehind, because we should be able to select any node as secondary and re-sync their state against primary. this flag allows for the two codepaths.